### PR TITLE
Original keeper added to `Keep`

### PR DESCRIPTION
**Pull request by andrewconner**
Will merge tonight, may cause a little bit of downtime. @ahsu1230

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/model/Keep.scala [hunk 1]
```diff
@@ -142,7 +142,7 @@ object Keep {
         "inDisjointLib" -> k.inDisjointLib,
         "urlId" -> k.urlId,
         "url" -> k.url,
-        "bookmarkPath" -> k.bookmarkPath,
+        "bookmarkPath" -> (None: Option[String]),
```
Comment (ahsu1230) [inline kifi-backend/modules/common/app/com/keepit/model/Keep.scala:145]: could you just remove this part if you're going to drop the `bookmarkPath` column?

**Comment by ahsu1230** [comment]
is it necessary to have an admin endpoint backfill the `originalKeeperId` field for all current keeps?

**Comment by andrewconner** [comment]
I'm actually going to just do it in SQL

**Comment by ahsu1230** [comment]
:+1:

**Pull request merged**